### PR TITLE
OSDOCS#6374: Update the GCP without workload identity section

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.13+

Issue: https://issues.redhat.com/browse/OSDOCS-6374

Link to docs preview: https://60808--docspreview.netlify.app/openshift-enterprise/latest/security/cert_manager_operator/cert-manager-authenticate-non-sts-gcp.html#cert-manager-configure-cloud-credentials-gcp-non-sts_cert-manager-authenticate-non-sts-gcp

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by xingxingxia** [comment]
lgtm from QE side
/lgtm

File: modules/cert-manager-configure-cloud-credentials-gcp-non-sts.adoc [hunk 1]
```diff
@@ -36,6 +36,15 @@ spec:
   serviceAccountNames:
   - cert-manager
 ----
++
+[NOTE]
+====
+The `dns.admin` role provides admin privileges to the service account for managing Google Cloud DNS resources. To ensure cert-manager runs with the service account that has the least privilege, you must create a custom role with the following permissions:
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply modules/cert-manager-configure-cloud-credentials-gcp-non-sts.adoc:42]: You probably don't need the _you must_ here. But, I'll leave that to your discretion.

**Comment by mburke5678** [comment]
@xenolinux A couple of small nits. Otherwise LGTM

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Pull request merged**